pallets/gear-built-in-actors/src/lib.rs

```diff
@@ -0,0 +1,443 @@
+// This file is part of Gear.
+
+// Copyright (C) 2021-2023 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! # Gear BuiltIn Actors Pallet
+//!
+//! The BuiltIn Actors pallet provides a a set of unique accounts that are treated as built-in
+//! actors ids (`ProgramId`).
+//!
+//! - [`Config`]
+//! - [`Call`]
+//!
+//! ## Overview
+//!
+//! The pallet implements a set of actors to handle messages from the message queue that
+//! require some sort of runtime-related logic to shortcut the usual messages processing flow.
+//! This provides an easy way for programs to interact with "pure runtime" actors thereby
+//! allowing contracts to build on top of functionality provided by the Runtime.
```

Comment:
It is better to use more common term `program`

```suggestion
//! allowing programs to build on top of functionality provided by the Runtime.
```